SYCL/ESIMD/api/functional/operators/operator_assignment.hpp

```diff
@@ -0,0 +1,96 @@
+//===-- operator_assignment.hpp - Functions for tests on simd assignment
+//      operators. --------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file provides functions for tests on simd assignment operators.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+// The test proxy is used to verify the move assignment was called actually.
+#define __ESIMD_ENABLE_TEST_PROXY
+
+#include "../common.hpp"
+#include "../shared_element.hpp"
+
+namespace esimd_test::api::functional::ctors {
```

Comment:
```suggestion
namespace esimd_test::api::functional::operators {
```

Reply:
I've applied this comment [here](https://github.com/intel/llvm-test-suite/pull/762/commits/92df51c1386dfd2e52709190f04f2c7baaec113d#diff-edf690369570284c343904e24bdf038b8c4bb8e6b620e467934162c5cdd1e05eR23)